sympy/calculus/singularities.py

```diff
@@ -32,3 +33,123 @@ def singularities(expr, sym):
                                   " implemented")
     else:
         return tuple(sorted(solveset(simplify(1/expr), sym)))
+
+###########################################################################
+###################### DIFFERENTIAL CALCULUS METHODS ######################
+###########################################################################
+
+
+def is_increasing(f, given_interval):
+    """
+    Returns if a function is increasing or not, in the given
+    `Interval`.
```

Comment:
It tells if a function is:

* increasing
* not increasing (can be anything: decreasing, (both increasing & decreasing) or neither)